frontend/javascripts/navbar.tsx

```diff
@@ -722,6 +724,7 @@ function AnonymousAvatar() {
         icon={<UserOutlined />}
         style={{
           marginLeft: 8,
+          marginTop: navbarHeight,
```

Comment:
does this fix an old problem or is it necessary for the new banner?

Reply:
This picture was supposed to show that it was a problem before, and I think you can tell from the picture. Anyway, I think I will fix this bug aswell (the black space it supposed to show an upcoming maintenance)
![image](https://github.com/scalableminds/webknossos/assets/57364771/3be8fb3d-aed1-414c-8cf9-ff8f28d40ebc)

Reply:
there we go, this is how it was before
![image](https://github.com/scalableminds/webknossos/assets/57364771/ca070b10-afd4-4a9b-a4c9-258694d59c42)

Reply:
> This picture was supposed to show that it was a problem before

> there we go, this is how it was before

I'm confused. Both pictures show the "before this PR" state?

Reply:
exactly.

Reply:
![image](https://github.com/scalableminds/webknossos/assets/57364771/b74f52d7-93ba-4864-a9d2-724758552caf)
this is how it looks now